src/ui/public/chrome/directives/app_switcher/app_switcher.js

```diff
@@ -1,10 +1,18 @@
 import DomLocationProvider from 'ui/dom_location';
 import { parse } from 'url';
-import { bindKey } from 'lodash';
+import { bindKey, filter } from 'lodash';
 import '../app_switcher/app_switcher.less';
 import uiModules from 'ui/modules';
 import appSwitcherTemplate from './app_switcher.html';
 
+function isNavLinkEnabled(link) {
+  return link.enable === undefined || link.enable;
+}
+
+function isNavLinkShown(link) {
+  return link.show === undefined || link.show;
```

Comment:
This is kind of silly, but I'm thinking if we want them to be shown and enabled by default, then why not switch this to use `hidden` and `disabled` terminology? I think the code might become a little bit clearer... what do you think?